lib/zendesk_apps_tools/command.rb

```diff
@@ -191,10 +191,12 @@ def update
       @command = 'Update'
 
       app_id = cache.fetch('app_id') || find_app_id
-      unless /\d+/ =~ app_id.to_s
+      app_url = "/api/v2/apps/#{app_id}.json"
```

Comment:
This API would indicate whether the app is [available for the account](https://developer.zendesk.com/rest_api/docs/core/apps#get-information-about-app), no? To determine if there's an installation we'd need to GET [`/api/v2/apps/installations/{id}.json`](https://developer.zendesk.com/rest_api/docs/core/apps#show-app-installation).

Reply:
@danielbreves 

> This API would indicate whether the app is available for the account, no?

Yes :)

A request to `/api/v2/apps/installations/` and searching for the app ID in the result would work as well. 

If the app is installed, the endpoint `/api/v2/apps/{app_id}.json` should exist/return a 200 (404 if not). The method employed in this PR makes us of this assumption, which I think is quicker (no need to search through a list of IDs).

Reply:
yes, Daniel is correct here.

Reply:
fair enough, if you no longer care if it's installed or not, that endpoint will work 👍

Reply:
...as long as you make the request using the account's subdomain and being authenticated, off course

Reply:
not a caching issue... this endpoint will return the app IF the app is available to you. that's what you want, right? to know if the id exists? What if you are trying to `zat update` an app that hasn't been installed yet? I think you can trust that endpoint. The only problem I see is if you have a wrong id in your .zat file which matches another existing app_id from an app that belongs to another account. In this case, the endpoint will NOT return 200 although the app would exist.